### PR TITLE
netdata-updater.sh now support -f to force re-installation

### PR DESCRIPTION
**Pull request by ktsaou**
even if git has not been updated

### CODE REVIEW

File: netdata-installer.sh [hunk 1]
```diff
@@ -673,7 +673,15 @@ if [ ${DONOTSTART} -eq 1 ]
         fi
         chmod 0664 "${NETDATA_PREFIX}/etc/netdata/netdata.conf"
     fi
-    echo >&2 "OK. It is now installed and ready."
+    cat >&2 <<"DONE1"
+
+
+  .-.   .-.   .-.   .-.   .-.   netdata
+     '-'   '-'   '-'   '-'   '  is installed on this system now!
```
Comment (paulfantom) [inline netdata-installer.sh]: Nice snake :smile:

**Comment by ktsaou** [comment]
It's a text chart :)

---
Sent from my iphone.

25 Σεπ 2016, 20:00, ο/η Paweł Krupa <notifications@github.com> έγραψε:

**Comment by ktsaou** [comment]
hm... snake you thought... I don't like snakes...

Should we do it like that:

```
  ^
  |.-.   .-.   .-.   .-.   .-.  netdata -.   .-.   .-.   .-.   .-.
  |   '-'   '-'   '-'   '-'   ' is installed and is running now!  '-'   
  +----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+----->
```

or like that:

```
  ^
  |.-.   .-.   .-.   .-.   .-.   
  |   '-'   '-'   '-'   '-'   '   netdata
  +----+-----+-----+-----+----->  is installed and is running now!
```

Which one do you prefer?

**Comment by paulfantom** [comment]
Second one is better.

**Pull request merged**